app/views/content/blog/my-first-week-as-a-trainee-teacher.md

```diff
@@ -0,0 +1,69 @@
+
+I enjoy the variety of information given during the morning events as it increases my understanding of the abilities and skills that schools are looking for when employing teachers.
+
+In the afternoon, I attend two subject seminars on the History of the British curriculum, which explain the differences and similarities between European and British secondary education.
+
+## Wednesday
+
+Today I attend several seminars on the practical side of teaching, such as marking, how to find sources and papers to prepare students for exams and techniques, how to adapt resources for different key stages and how to plan lessons according to the curriculum.
+
+I enjoy these seminars because they are more practical and give me the chance to apply and put into practice what I already know, as well as learn more about the curriculum of the different key stages.
+
+## Thursday
+
+This morning, all the students enrolled on the course attend an informative lecture on the different master components that are part of the PGDE, as well as information about school placements.
+
+Similar to previous days, my lessons continue in the afternoon with a session focused on preparing us for the group master assignment. We are divided into small groups of four and asked to consult a newspaper article focused on the topic: ’issues in education’. We then work together on a 10-minute presentation to deliver to the rest of the class.
+
+I don’t enjoy this activity because I am nervous about presenting in front of so many people, especially on a topic that I don’t know much about. However, even though I am nervous, the presentation goes well, and I overcome the challenge with the support of my group, who encourage me throughout the presentation.
+
+Reflecting on this experience, I admit that this practice presentation was a good way to prepare for my upcoming school placements where I will be presenting and leading lessons.
+
+## Friday
+
+My day starts differently to previous days. I spend the morning at home reading an article and watching a pre-recorded seminar on children and poverty. I then read some papers and watch two short documentaries on education and poverty. After completing these tasks, I met my subject group on Google Meet and we spent the rest of the morning discussing what we have seen.
+
+In the afternoon I attended a seminar on GCSE and A-level assessment. In this session we worked in a group to identify and discuss the different assessment specifications for languages.
+
+The group work is important because it aimed to familiarise us with the different subjects’ language specifications and syllabus, particularly interesting for me as I didn’t attend school in the UK, so learning about specifications and the curriculum was exciting to me.
```

Comment:
```suggestion
The group work is important because it aimed to familiarise us with the different subjects’ language specifications and syllabus, particularly interesting for me as I didn’t attend school in the UK, so learning about specifications and the curriculum is exciting to me.
```